ircDDBGateway/APRSTransmit/APRSTransmitAppD.cpp

```diff
@@ -79,8 +79,15 @@ int main(int argc, char** argv)
 	}
 
 	wxString repeater = parser.GetParam(0U);
+	if (repeater.length() != LONG_CALLSIGN_LENGTH) {
+		wxString callErrorMsg;
+		callErrorMsg << wxT("Invalid repeater call. ") << repeater << wxT("Call must be ") << LONG_CALLSIGN_LENGTH << wxT(" characters long.\n");
```

Comment:
> On May 9, 2016, at 9:04 PM, Geoffrey Merck <notifications@github.com> wrote:
> 
> In ircDDBGateway/APRSTransmit/APRSTransmitAppD.cpp:
> 
> > @@ -79,8 +79,15 @@ int main(int argc, char** argv)
> >  	}
> >  
> >  	wxString repeater = parser.GetParam(0U);
> > +	if (repeater.length() != LONG_CALLSIGN_LENGTH) {
> > +		wxString callErrorMsg;
> > +		callErrorMsg << wxT("Invalid repeater call. ") << repeater << wxT("Call must be ") << LONG_CALLSIGN_LENGTH << wxT(" characters long.\n");
> 
> Hi, It looks like I missed that one, Thanks for the hint. wxS shall be used as I understand the documentation.

My understanding is that it’s not necessary to do anything anymore.  Everything is unicode.  The wxS documentation says:

The use of this macro is optional as the translation will always be done at run-time even if there is a mismatch between the kind of the literal used and the string or character type used in the current build. However using it can be beneficial in performance-sensitive code to do the conversion at compile-time instead.

Premature optimization is evil.  We probably shouldn’t bother with wxS anywhere unless there’s a demonstrated need for it. 

--
Jeremy McDermond (NH6Z)
Xenotropic Systems
mcdermj@xenotropic.com